### PR TITLE
LPS-130473 Audit and refactor tests for component Tags tests to use SACRED model | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-130473

cc @david-gutierrez-mesa 

Hi @Tim-Cao, I removed the follow cases below because they were duplicated. After these changes get in master I'm still going to send other PR refactoring some macros on TagsNavigationPortlet.macro.

RemoveTagViaBlogs -> Duplicated by ViewRemovedTagViaBlogs
AddTagViaBlogs -> Duplicated by AddSingleTagViaBlogs
AddSelectTagViaBlogs -> Duplicated by AddMultipleTagsViaBlogs

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
Hi @ozysouza, we shouldn't refactor other team macros, for example https://github.com/liferay-echo/liferay-portal/pull/7293/commits/efaa9b226c4bc72a79c8872de57bc9a988248953
and https://github.com/liferay-echo/liferay-portal/pull/7293/commits/ea2aee0aeddee18cee72fdfc4bb294d086720771

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/tags/Tag.macro [hunk 1]
```diff
@@ -87,6 +87,11 @@ definition {
 	}
 
 	macro mergeTags {
+		Select(
+			key_selectFieldLabel = "Into This Tag",
+			locator1 = "Select#ANY",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/tags/Tag.macro:113]: Select#GENERIC_SELECT_FIELD

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/tags/Tag.macro [hunk 2]
```diff
@@ -61,6 +61,27 @@ definition {
 		Alert.viewRequiredField();
 	}
 
+	macro deleteCP {
+		PortletEntry.deleteViaMenuItem(rowEntry = "${tagName}");
+
+		Alert.viewSuccessMessage();
+	}
+
+	macro editMenu {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/tags/Tag.macro:70]: macro editCP

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/tags/Tag.macro [hunk 3]
```diff
@@ -61,6 +61,27 @@ definition {
...
+			menuItem = "Edit",
+			rowEntry = "${tagMenuName}");
+
+		PortletEntry.inputName(name = "${tagNameEdit}");
+
+		if (!(isSet(configureWithoutSave))) {
+			PortletEntry.save();
+		}
+		else {
+			Button.clickCancel();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/tags/Tag.macro:81]: Extract a single macro for cancel, named cancelEdit

**Review by Tim-Cao — COMMENTED** [review]
Could you update according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, we shouldn't refactor other team macros, for example [efaa9b2](https://github.com/liferay-echo/liferay-portal/commit/efaa9b226c4bc72a79c8872de57bc9a988248953) and [ea2aee0](https://github.com/liferay-echo/liferay-portal/commit/ea2aee0aeddee18cee72fdfc4bb294d086720771)

Oh okay! I did it because the logic would be better but I'll remove my changes.

**Comment by ozysouza** [comment]
ci:close

**Pull request closed without merging**